CAIPs/caip-211.md

```diff
@@ -18,7 +18,12 @@ within one) and RPC endpoints (i.e. preferential routing for specific nodes).
 
 ## Abstract
 
-
+Without a profile of this CAIP which defines implicit values for a namespace,
+setting the `rpcDocuments` and `rpcEndpoints` values in the `scopeObject`s of a
+[CAIP-25][] negotiation does not make much sense. Once those implicit values
```

Comment:
I would argue this to not be true, because a rpc document could be introducing new methods and not necessarily overriding implicit values.

Reply:
Good point--- I'm trying to telegraph that this introduces potential security risks (since an explicit one COULD override an implicit one) so using this property is a little riskier until/unless a profile has been written... will rewrite in the meantime, in any case